app/Http/Controllers/Legacy/Web/ActionsController.php

```diff
@@ -113,9 +150,15 @@ public function update(Request $request, Action $action)
      */
     public function destroy(Action $action)
     {
-        $action->delete();
+        $trashed = $action->delete();
+
+        if ($trashed) {
+            // Log that an action was deleted.
+            info('action_deleted', ['id' => $action->id]);
+
+            return $this->respond('Action deleted.', 200);
+        }
```

Comment:
Do you think this way would be more readable? Totally up to you!
```
if (! $trashed) {
    return response()->json(['code' => 500, 'message' => 'There was an error deleting the action']);
}

// Log that an action was deleted.
info('action_deleted', ['id' => $action->id]);

return $this->respond('Action deleted.', 200);
```

Reply:
omg yes this is so much better! Updated!